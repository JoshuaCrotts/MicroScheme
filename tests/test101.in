;
; Setting a variable of a different type than it previously was.
;

(define square (lambda (x) (* x x)))
(define y square)
(y 20)
(square 20)
(set! y 50)
y
(square 20)
(set! square "square")
square