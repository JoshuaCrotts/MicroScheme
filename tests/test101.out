400
400
50
400
"square"
